package.json

```diff
@@ -15,7 +15,8 @@
       "types/broccoli-concat",
       "types/ember-cli-htmlbars",
       "tests/scenarios",
-      "tests/app-template"
+      "tests/app-template",
+      "tests/addon-template"
```

Comment:
Should these be versioned (e.g. `app-template-3-24`)? It seems reasonable to have a template for each of our supported LTS versions (for example). That allows us to test against the ember-cli blueprint's of those supported LTS's.

Can you describe the expected workflow / process for updating these? Do we update them for every ember-cli release? Some other cadence? It would be great to start fleshing out our `CONTRIBUTING.md` with these kinds of details...

Reply:
> Can you describe the expected workflow / process for updating these?

We would run the following probably after every new LTS is released.

1.  Run ember-cli-update in the template directories to pull in the latest template changes.
2. Update 'tests/scenarios.ts' to link to make sure that our support matrix is correct.


> Should these be versioned

We can but we would have to restructure the test infra a bit if we want to. Right now we are just selectively installing the ember-cli and ember-source deps within this base template but if we feel like the template structure from one to the other is worth testing we can. If we go this route then we might want to rethink the above update workflow.

Reply:
👍  - Sounds good. Mind throwing a small section into `CONTRIBUTING.md` with that "template update" plan?

Reply:
Added